lit_tests/kernel/wave/integration.py

```diff
@@ -0,0 +1,309 @@
+# RUN: python %s | FileCheck %s
+
+import iree.turbine.aot as aot
+import torch
+import textwrap
+from jinja2 import Environment, BaseLoader
+import iree.turbine.kernel as tk
+import iree.turbine.kernel.lang as tkl
+import iree.turbine.kernel.wave as tkw
+from iree.turbine.kernel.lang.global_symbols import *
+from iree.turbine.kernel.wave.utils.general_utils import (
+    run_test,
+)
+from iree.turbine.kernel.wave.templates.vanilla_attention import (
+    get_bhsd_attention_kernel,
+)
+from iree.turbine.kernel.wave.templates.attention_common import (
+    AttentionShape,
+)
+from iree.turbine.kernel.wave.scheduling.schedule import SchedulingType
+from iree.turbine.kernel.wave.compile import WaveCompileOptions, wave_compile
+from iree.turbine.kernel.wave.utils.general_utils import (
+    get_default_scheduling_params,
+)
+from iree.turbine.kernel.wave.utils.run_utils import (
+    set_default_run_config,
+)
+from iree.turbine.kernel.wave.utils.compile_utils import (
+    get_wave_module_body_asm,
+    get_kernel_name,
+)
+from iree.turbine.runtime.op_reg.base import (
+    CustomOp,
+    KernelSelection,
+    KernelBuilder,
+)
+from iree.turbine.support.ir_imports import (
+    PassManager,
+)
+from iree.turbine.runtime.op_reg.impl_helper import (
+    call_function,
+)
+from iree.compiler.ir import (
+    Module,
+    Context,
+    RankedTensorType,
+    Operation,
+    MLIRError,
+)
+from iree.turbine.transforms.merger import Merger
+from typing import Optional
+
+_JINJA2_ENVIRONMENT: Optional[Environment] = None
+
+
+def _get_jinja2_env() -> Environment:
+    global _JINJA2_ENVIRONMENT
+    if _JINJA2_ENVIRONMENT is None:
+        _JINJA2_ENVIRONMENT = Environment(loader=BaseLoader())
+    return _JINJA2_ENVIRONMENT
+
+
+@CustomOp.register()
+class WaveBhsdFlashAttention(CustomOp):
```

Comment:
Maybe we should call this filename or function name to indicate that this is indeed integration for sharktank

Reply:
I do see `test_wave_aot_integration`, which is the right direction and would be even gooder to call out sharktank :D